deployments/rpm/deployment.sh

```diff
@@ -6,7 +6,7 @@ _this_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
 _repo_root=$(realpath "${_this_dir}/../..")
 
-_package_name="veraison"
+_package_name="veraison-server"
```

Comment:
`veraison-services` is probably a better name since the package includes multiple servers (verification, provisioning, etc). Also, this should be consitent across all packages, since we don't want the package name on Fedora to be different from the one on Ubuntu (so please update `debian` accordingly).